src/EnergyPlus/DataSurfaces.hh

```diff
@@ -320,89 +321,59 @@ namespace DataSurfaces {
-    constexpr int WSC_ST_SwitchableGlazing(2);
-    constexpr int WSC_ST_ExteriorShade(3);
-    constexpr int WSC_ST_InteriorBlind(4);
-    constexpr int WSC_ST_ExteriorBlind(5);
-    constexpr int WSC_ST_BetweenGlassShade(6);
-    constexpr int WSC_ST_BetweenGlassBlind(7);
-    constexpr int WSC_ST_ExteriorScreen(8);
-
-    // WindowShadingControl Control Types
-    constexpr int WSCT_AlwaysOn(1);                       // AlwaysOn
-    constexpr int WSCT_AlwaysOff(2);                      // AlwaysOff
-    constexpr int WSCT_OnIfScheduled(3);                  // OnIfScheduleAllows
-    constexpr int WSCT_HiSolar(4);                        // OnIfHighSolarOnWindow
-    constexpr int WSCT_HiHorzSolar(5);                    // OnIfHighHorizontalSolar
-    constexpr int WSCT_HiOutAirTemp(6);                   // OnIfHighOutsideAirTemp
-    constexpr int WSCT_HiZoneAirTemp(7);                  // OnIfHighZoneAirTemp
-    constexpr int WSCT_HiZoneCooling(8);                  // OnIfHighZoneCooling
-    constexpr int WSCT_HiGlare(9);                        // OnIfHighGlare
-    constexpr int WSCT_MeetDaylIlumSetp(10);              // MeetDaylightIlluminanceSetpoint
-    constexpr int WSCT_OnNightLoOutTemp_OffDay(11);       // OnNightIfLowOutsideTemp/OffDay
-    constexpr int WSCT_OnNightLoInTemp_OffDay(12);        // OnNightIfLowInsideTemp/OffDay
-    constexpr int WSCT_OnNightIfHeating_OffDay(13);       // OnNightIfHeating/OffDay
-    constexpr int WSCT_OnNightLoOutTemp_OnDayCooling(14); // OnNightIfLowOutsideTemp/OnDayIfCooling
-    constexpr int WSCT_OnNightIfHeating_OnDayCooling(15); // OnNightIfHeating/OnDayIfCooling
-    constexpr int WSCT_OffNight_OnDay_HiSolarWindow(16);  // OffNight/OnDayIfCoolingAndHighSolarOnWindow
-    constexpr int WSCT_OnNight_OnDay_HiSolarWindow(17);   // OnNight/OnDayIfCoolingAndHighSolarOnWindow
-    constexpr int WSCT_OnHiOutTemp_HiSolarWindow(18);     // OnIfHighOutsideAirTempAndHighSolarOnWindow
-    constexpr int WSCT_OnHiOutTemp_HiHorzSolar(19);       // OnIfHighOutsideAirTempAndHighHorizontalSolar
-    constexpr int WSCT_OnHiZoneTemp_HiSolarWindow(20);    // OnIfHighZoneAirTempAndHighSolarOnWindow
-    constexpr int WSCT_OnHiZoneTemp_HiHorzSolar(21);      // OnIfHighZoneAirTempAndHighHorizontalSolar
-
     // WindowShadingControl Slat Angle Control for Blinds
-    constexpr int WSC_SAC_FixedSlatAngle(1);
-    constexpr int WSC_SAC_ScheduledSlatAngle(2);
-    constexpr int WSC_SAC_BlockBeamSolar(3);
+    enum class WindowShadingControlSlatAngleControl {
+        Invalid = -1,
+        FixedSlatAngle,
+        ScheduledSlatAngle,
+        BlockBeamSolar,
+        Num
+    };
 
     // Parameter for window screens beam reflectance accounting
-    constexpr int DoNotModel(0);
-    constexpr int ModelAsDirectBeam(1);
-    constexpr int ModelAsDiffuse(2);
-
-    // Parameters for window divider type
-    constexpr int DividedLite(1);
-    constexpr int Suspended(2);
+    enum class ScreenBeamReflectanceAccounting
```

Comment:
I would change `Accounting` to `Model` and then remove `ModelAs` from the enumeration values. Generally, try to reduce redundancy between enumeration type name and enumeration value names.